rusk-wallet/src/bin/command.rs

```diff
@@ -176,7 +176,7 @@ pub(crate) enum Command {
     PhoenixWithdraw {
         /// Profile index for the shielded address from which to make the
         /// withdraw request [default: 0]
-        #[clap(short, long)]
+        #[clap(short = 'a', long)]
```

Comment:
Be consistent (this is "a", the others are "s")


Btw, I would remove the short version and use only the long